### PR TITLE
Configuration tab

### PR DESCRIPTION
**Pull request by nerimartinez**

### CODE REVIEW

File: ui/src/components/DeploymentConfigurationTab.vue [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        },
+      ],
+    };
+  },
+
+  methods: {
+    prepareRow,
+    getFileSize,
+    downloadFile(url) {
+      const downloadUrl = `https://vendor.cnct.info/public/v1/media/folders/accounts/${this.accountId}/${this.deploymentId}/pprs/files/MFL-9618-6980-7316`;
```
Comment (arnaugiralt) [inline ui/src/components/DeploymentConfigurationTab.vue:140]: Could we get this URL in some other way? Hostname is hardcoded right now.

File: ui/src/components/DeploymentConfigurationTab.vue [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+      downloader({ url: downloadUrl });
+    },
+    downloader,
```
Comment (arnaugiralt) [inline ui/src/components/DeploymentConfigurationTab.vue:144]: No need to expose downloader to the template

File: ui/src/components/DeploymentConfigurationTab.vue (hunk 1 above)
Comment (nerimartinez) [reply ui/src/components/DeploymentConfigurationTab.vue:140]: so far, this is hand made, after the upload is implemented, this is gonna be changed

File: ui/src/components/DeploymentConfigurationTab.vue (hunk 2 above)
Comment (nerimartinez) [reply ui/src/components/DeploymentConfigurationTab.vue]: you are right!

**Review by arnaugiralt — APPROVED** [review]

**Pull request merged**